### ISSUE TITLE
"Add X Fastener" commands redundant

### ISSUE THREAD

**Issue by hasecilu** [shaise/FreeCAD_FastenersWB#355]
Currently the description appears 2 times and the norm 3 times.

I think it's better to remove the description from the `Menutext` also I'm not sure if include again the norm in the `ToolTip` and don't know if the best place for the verb "Add" is the `MenuText` or the `ToolTip`.

Finally, the commands from main toolbar have "Fasteners_" prefix, the fasteners commands have "FS" maybe update that also

![image](https://github.com/shaise/FreeCAD_FastenersWB/assets/53124818/22bb717b-e888-43e0-9b74-fd113db96c84)

**Comment by shaise** [comment]
I dont think we can change the FS<some-fastener> command. I think it will break loading of old files with fasteners.

**Comment by hasecilu** [comment]
Didn't think about that, apart from that there is no other reason to change the command because there are no Wiki pages for that.

Only is left to discuss about MenuText and ToolTip. It seems that only the ToolTip has a way to shrink long string but MenuText not. I was thinking on having it more compact (visually).

![image](https://github.com/shaise/FreeCAD_FastenersWB/assets/53124818/5edfdb5c-e4f2-4541-a475-ad971eb91c64)

It seems even for Russian special cases for singular/plural is used.

**Comment by shaise** [comment]
yes, definitely we need to remove the redundancy. should look like your second image. But it should not affect the pull down mode:

![image](https://github.com/shaise/FreeCAD_FastenersWB/assets/5813568/bb87dccf-defd-4964-8c80-6ad00f188340)
